firmware/controllers/settings.cpp

```diff
@@ -1159,30 +1145,56 @@ void initSettings(void) {
 
 #if HAL_USE_ADC
 	addConsoleActionSS("set_analog_input_pin", setAnalogInputPin);
-#endif
+#endif // HAL_USE_ADC
 	addConsoleActionSS(CMD_LOGIC_PIN, setLogicInputPin);
 	addConsoleActionI("set_pot_spi", setPotSpi);
-#endif /* EFI_PROD_CODE */
+#endif // EFI_PROD_CODE
+}
+
+void printDateTime() {
+#if EFI_RTC
+	printRtcDateTime();
```

Comment:
This can't really fail anymore, no more ...

```cpp
time_t unix_time = GetTimeUnixSec();
	if (unix_time == -1) {
		efiPrintf("invalid time in RTC cell");
```

should we validate ChibiOS's `rtcGetTIme` somehow?  docs indicate it can't really fail either; no signalling about this case